sdk-python/awaazde/check_message_created.py

```diff
@@ -0,0 +1,63 @@
+import argparse
+import csv
+from datetime import datetime
+
+from constants import CommonConstants
+from utils import ADMessageUtils
+
+message_data = ''
+
+
+def parseArguments():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("username", help="Username", type=str)
+    parser.add_argument("password", help="Password", type=str)
+    parser.add_argument("organization", help="Organization ", type=str)
+    parser.add_argument("path", help="Path for the csv ", type=str)
+    parser.add_argument("-f", "--fields", help="Filters Used for checking messages ", nargs="*", type=str,
+                        default="send_on")
+    parser.add_argument("-v", "--values", help="Value for filters for checking messages ", type=str,
+                        default=datetime.now().strftime(CommonConstants.DEFAULT_DATE_FORMAT))
+    args = parser.parse_args()
+    return args
+
+
+def drop_messages_to_file(message_data, path, file_name):
```

Comment:
@ashwini-ad  Can we add this as utility method ? to create file form messages data ? this code is getting repeated 2 times